src/app_recovery.js

```diff
@@ -194,6 +199,16 @@ class AppModeApp extends LitElement {
   _nextStep = () => {
     this.isLoggedIn = this.context.store.networkContext.token;
     this.activeStepNumber++;
+    
+    // Set setup as completed when reaching the DONE step
+    if (this.activeStepNumber === STEP_DONE) {
+      store.updateState({
+        setupContext: {
+          ...this.context.store.setupContext,
+          isCompleted: true
```

Comment:
Hmm, this is going to persist in cookies and break dev setup the next time you drop into recovery mode?